src/test/java/core/basesyntax/service/impl/TransactionParserImplTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.TransactionParser;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TransactionParserImplTest {
+    private static TransactionParser transactionParser;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        transactionParser = new TransactionParserImpl();
+    }
+
+    @Test
+    public void parseTransactionList_Ok() {
+        List<String> stringList = new ArrayList<>();
+        stringList.add("b,banana,20");
+        stringList.add("b,apple,100");
+        stringList.add("s,banana,100");
+        stringList.add("p,banana,13");
+        stringList.add("r,apple,10");
+        stringList.add("p,apple,20");
+        stringList.add("p,banana,5");
+        stringList.add("s,banana,50");
```

Comment:
can we move it to some constant var like `TEST_DATA` and initialize it in the `before each`?
we can also use `List.of()` method instead, same with the `expectedList` var